desktop/enterprise/release-notes.md

```diff
@@ -17,12 +17,26 @@ For information on system requirements, installation, and download, see:
 
 For Docker Enterprise Engine release notes, see [Docker Engine release notes](/engine/release-notes).
 
+## Version 2.3.0.0
+2020-04-22
+
+Docker Desktop Enterprise 2.3.0.0 contains a Kubernetes upgrade. Note that your local Kubernetes cluster in Version Pack 3.0 will be reset after installing this version.
+
+This version contains all the bug fixes and minor changes between Docker Desktop Community versions 2.1.0.1 and 2.3.0.0 included, see:
```

Comment:
Minor rephrase: "This version includes all the bug fixes and minor changes between Docker Desktop Community versions 2.1.0.1 and 2.3.0.0. For details see:"

Reply:
Done, thanks!